docs/index.html

```diff
@@ -0,0 +1,156 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Ibhicheckesho | Home</title>
+  </head>
+  <body>
+    <header id="header-home">
+      <div class="container">
+        <nav id="main-nav">
+          <div class="container">
+            <div id="logo">
+              <a href="index.html"
+                ><img src="./images/logo.png" alt="Watubaki-logo"
+              /></a>
+            </div>
+            <div class="baga-icon-container">
+              <input id="baga-toggler" type="checkbox" />
+              <div class="baga-icon">
+                <div></div>
+              </div>
+            </div>
+            <ul class="nav-menu">
+              <li><a class="nav-menu-link" href="./index.html">Home</a></li>
+              <li><a class="nav-menu-link" href="./about.html">About</a></li>
+            </ul>
+          </div>
+        </nav>
+        <section id="header-content">
+          <h1>
+            <span id="header-content-span">Hello! W'anna Lough? Welcome</span
+            ><span id="header-content-span1">Watubaki</span
+            ><span id="header-content-span2">Standup Comedy</span
+            ><span id="header-content-span3">March 2022</span>
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum
+            quibusdam similique tenetur obcaecati maxime, molestias
+            exercitationem. Eaque voluptatum optio, tempora sit recusandae error
+            tenetur minus ipsam laboriosam? Ea, alias. Pariatur?
+          </p>
+          <hgroup>
+            <h2>2022.10.15(THU) ~ 16(FRI)</h2>
+            <h3>@Mlimani City Shopping Center.</h3>
+          </hgroup>
+        </section>
+      </div>
+    </header>
+    <main id="home-main">
+      <section id="schedule" class="main-section">
+        <div class="container">
+          <h2>Schedule <span></span></h2>
+          <div class="cards">
+            <section class="card">
+              <i class="fas fa-anchor"></i>
+              <h3>Opening</h3>
+              <p class="shedule-p">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quo
+                sed sapiente, voluptatibus labore fugit repellat?
+              </p>
+            </section>
```

Comment:
- [ ] For this section the background is looking okay but you can improve it to look like the design check images down:

Design
![og](https://user-images.githubusercontent.com/44745855/132837558-b3e87012-6b81-43f7-9b74-dd6a88f8adab.png)

your page

![cap2](https://user-images.githubusercontent.com/57562869/152236384-8a621f68-7cd1-41d5-be37-dd585a6b2e10.png)

Reply:
I could not get the image, is this supposed to be mandatory?